themes/bootstrap5/templates/ajax/itemLocationList.phtml

```diff
@@ -0,0 +1,12 @@
+<?php $availabilityStatus = $this->availabilityStatus(); ?>
+<?php foreach ($this->locationList as $location): ?>
+  <div class="groupLocation">
+    <span class="<?= $availabilityStatus->getClass($location['availability'])?>">
```

Comment:
Should we `escapeHtmlAttr` here just in case getClass returns something weird?

Reply:
I added the escape function.